### PR TITLE
WAVE2-914 - handle grad vs undergrad compare program page url

### PR DESCRIPTION
**Pull request by balrajb**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<div class="<?php echo $classes; ?> filters">
+  <div class="row">
+    <div class="container">
+      <div class="row">
+
+        <form id="filters" role="form">
+          <div class="col-md-3 vspacer-xs vspacer-sm degree">
+            <p><?php print t('Filter by degree'); ?>:</p>
+            <p>
+              <button type="button" class="btn btn-default masters" value="masters">Masters</button>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc]: Change Masters to Master

File: sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        </div>
+      </div>
+      <div class="row">
+
+        <div class="col-sm-12 results">
+          <h3>
+            <?php print t('Results for...'); ?>
+          </h3>
+          <div id="filter-results"><!-- Keyword filters appear here
+            --><span class="label label-default masters">Masters<span class="glyphicon glyphicon-remove" aria-hidden="true"></span></span><!--
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc]: Masters to Master

File: sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+      $("#filter-results").on("click", ".label .glyphicon", function() {
+        // With a filter removed, we need to update the display state...
+        var $label = $(this).parent();
+        if ($label.hasClass("keyword")) {
+          // If it is a search string being removed, we'll need to refresh the view/page
+          $(this).parent().remove();
+        }
+        else {
+          // Other types of filters
+          if ($label.is(".masters, .phd")) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: Masters to Master

File: sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+          updateProgramList();
+        }
+      });
+
+
+      // Show/hide filter labels in results region
+
+      function updateFilterBox() {
+        $(".masters,.phd,.course-based,.thesis-based","#filter-results").hide();
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: Masters to Master

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+?>
+<div class="col-xs-12 col-sm-6 col-md-4 program">
+	<div class="rounded-tile program-tile-large">
+        <button class="btn btn-rounded compare-program" data-id="<?php echo $fields['nid']->raw; ?>" type="button">
+            <span class="sr-only">Toggle Compare</span>
+            <span class="glyphicon glyphicon-star-empty"></span>
+        </button>
+        <!-- Check if Masters is matched tid name, if so add an 's' for the filter to work -->
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php]: This rewrite of Master to Masters will no longer be needed once above code has been changed.

**Comment by balrajb** [comment]
Changed Masters to Master in https://github.com/ucalgary/www.ucalgary/pull/545/commits/95b6457f6f1037f9e09f98be5b8655ac1447c8a5

**Comment by dpfitzsi** [comment]
Merged in to wave2-910-912-914-974-fe-compare branch.

**Pull request closed without merging**